homework/Vasili_Zasinets/AbstractMath/src/main/java/com/tutrit/java/quickstart/abstractmath/MathOperationRepository.java

```diff
@@ -0,0 +1,16 @@
+package com.tutrit.java.quickstart.abstractmath;
+
+import java.util.List;
+
+public class MathOperationRepository {
+
+    public static final List<MathOperation> mathOperationList = List.of(
+            new DivisionOperation(),
+            new SubtractionOperation(),
+            new MultiplicationOperation(),
+            new AdditionOperation()
+    );
+
+    private MathOperationRepository() {
+    }
+}
```

Comment:
Many older tools misbehave(плохо себя вести) unless the last line of data in the text file ends with a newline or carriage return / newline combination. They ignore this line because it ends with ^ Z (eof) instead. This can lead to data loss.